content/CORETutorials/JailsPluginsVMs/Plugins/MinIOPlugin.md

```diff
@@ -0,0 +1,208 @@
+
+Enter a name for the plugin in **Jail Name** and adjust the networking settings as needed.
+You can use the default [Network Address Translation (**NAT**)](https://datatracker.ietf.org/wg/nat/about/), enable **DHCP**, or manually define IP addresses.
+
+{{< hint type=tip >}}
+If migrating data from an existing S3 service deployment, ensure ports for the Minio plugin are different from the existing service.
+
+{{< /hint >}}
+
+Click **Save** to install.
+A dialog confirms when the installation completes and shows post-install notes, including the `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`.
+You can view the post-install notes later by expanding the entry for the installed plugin in **Plugins** and clicking <i class="fa fa-file-alt" aria-hidden="true" title="File"></i> **Post Install Notes**.
+
+The **Plugins** screen shows the installed plugin.
+
+{{< trueimage src="images/CORE/13.0/MinioPluginInstalled.png" alt="Minio Plugin Installed" id="Minio Plugin Installed" >}}
+
+Click <i class="material-icons" aria-hidden="true" title="Expand">chevron_right</i> to expand the Minio plugin details and management options.
+Click <i class="fa fa-stop" aria-hidden="true" title="Stop"></i>&nbsp;**STOP** to stop the jail before making any changes.
+
+Click <span class="material-icons">device_hub</span>&nbsp;**MOUNT POINTS** and follow the instructions in [Setting Up Jail Storage]({{< relref "SettingUpJailStorage.md" >}}) to mount the destination dataset you created in [First Steps](#first-steps).
+
+Click <span class="material-icons">play_arrow</span>&nbsp;**START** to restart the plugin and then click <span class="material-icons">settings</span>&nbsp;**MANAGE** to go to the **MinIO Console** and log in.
+
+{{< trueimage src="images/CORE/13.0/MinioPluginConsole.png" alt="MinIO Console" id="MinIO Console" >}}
+
+## Migrating from S3 Service to Minio Plugin
+
+After completing the initial install of the Minio plugin, users with an existing S3 service deployment need to migrate S3 data from the previous service deployment to the new plugin.
+Migrating from the built-in S3 service to the Minio plugin could require an extended service window.
+
+{{< enterprise >}}
+Beginning in CORE 13.0-U6, Enterprise customers with the S3 service running or enabled are prevented from upgrading to the next major version.
+
+TrueNAS Enterprise customers are strongly encouraged to contact iXsystems Support for assistance with the migration process.
+
+{{< expand "Contacting Support" "v" >}}
+{{< include file="content/_includes/iXsystemsSupportContact.md" >}}
+{{< /expand >}}
+{{< /enterprise >}}
+
+### Migration Overview
+Users with the S3 service enabled must upgrade CORE to version **13.0-U6** and complete the migration process **before** upgrading to a later major version.
+
+The migration path involves [installing a MinIO Client](#installing-the-minio-client) (`mc`) release with the required feature and function support to migrate the S3 service deployment.
```

Comment:
In this instance mc is not a command, it is used as part of a command but not in this instance so the emphasis is incorrect. 
```suggestion
The migration path involves [installing a MinIO Client](#installing-the-minio-client) (MC) release with the required feature and function support to migrate the S3 service deployment.
```